eng/build-test.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
Who calls these `build-test` scripts?

Reply:
The CI scripts currently do. I originally removed them but I wasn't able to easily change the project without a wrapper script so I decided to keep them, at least for now.